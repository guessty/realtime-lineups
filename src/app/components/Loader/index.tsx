import * as React from 'react'
//
import StyledLoader from './StyledLoader'

const LOADING = 'Loading...'

export default () => (
  <React.Fragment>
    <StyledLoader>
      <h1>{LOADING}</h1>
    </StyledLoader>
  </React.Fragment>
)
