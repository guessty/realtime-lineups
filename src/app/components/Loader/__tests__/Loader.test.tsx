import * as React from 'react'
import * as Enzyme from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'
import * as renderer from 'react-test-renderer'
//
import Loader from './../index'

Enzyme.configure({adapter: new Adapter()})


describe('<Loader />', () => {
  let component: any
  let componentWrapper: any

  beforeEach(() => {
    component = (
      <Loader />
    )
    componentWrapper = Enzyme.shallow(component, {})
  })

  it('should be defined', () => {
    expect(Loader).toBeDefined()
  })

  it('should render', () => {
    expect(componentWrapper.length).toEqual(1)
  })
  
  it('capturing Snapshot of Loader', () => {
    const renderedValue =  renderer.create(component).toJSON()
    expect(renderedValue).toMatchSnapshot();
  })
})

