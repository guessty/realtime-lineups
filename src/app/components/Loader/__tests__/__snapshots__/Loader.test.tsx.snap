// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> capturing Snapshot of Loader 1`] = `
<div
  className="sc-bdVaJa khUSez"
>
  <h1>
    Loading...
  </h1>
</div>
`;
